src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -1017,6 +1013,30 @@ public Long getValue()
         return cfGauge;
     }
 
+    /**
+     * Same as {@link #createTableGauge(String, String, Gauge, Gauge)} but includes the ability to include a deprecated
+     * name for a table {@code Gauge}. Prefer that method when deprecation is not necessary.
+     *
+     * @param deprecated The name of the deprecated metric cannot be {@code null}
+     */
+    protected <G,T> Gauge<T> createTableGauge(String name, String alias, String deprecated, Gauge<T> gauge,
```

Comment:
Is there any caller of this new `createTableGauge()` overload where `name` and `alias` aren't identical.